src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.product {
```

Comment:
`product` looks a bit misleading name (product**s** or catalog, or just main, etc). Also, according to BEM rules shouldn't in this file